### PR TITLE
Correct is_function behavior 2

### PR DESCRIPTION
**Pull request by 0x0f0f0f**

### CODE REVIEW

**Comment by 0x0f0f0f** [comment]
@Roger-luo ready. Could you please bump to 0.10.8 ?

File: src/analysis/split.jl [hunk 1]
```diff
@@ -6,18 +6,18 @@ Split doc string from given expression.
...
+        return line, doc, expr
 
         # quote
         #     @doc "" struct <name> end
         # end
         @case Expr(:block, ::LineNumberNode, stmt)
-            line, doc, expr = split_doc(stmt)
-            return line, doc, expr
+        line, doc, expr = split_doc(stmt)
+        return line, doc, expr
```
Comment (Roger-luo) [inline src/analysis/split.jl:18]: I guess you run JuliaFormatter? It doesn't recognize MLStyle's match table unfortunately...

**Review by Roger-luo — CHANGES_REQUESTED** [review]
need to remove those indent changes

**Comment by 0x0f0f0f** [comment]
> need to remove those indent changes



> need to remove those indent changes

:facepalm: my bad

**Pull request closed without merging**